### PR TITLE
feat(Player Page): Add support for mediaSession

### PR DESCRIPTION
**Pull request by lavalleeale**
**What is the purpose of this pull request? (put an "X" next to item)**

- [ ] Documentation update
- [ ] Bug fix
- [X] New feature
- [ ] Other, please explain:

**What changes did you make? (Give an overview)**
Added support for the chrome `mediaSession` API so that hardware play/pause/next/back buttons will work to control the player
**Which issue (if any) does this pull request address?**
N/A

### CODE REVIEW

File: src/renderer/pages/player-page.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const React = require('react')
 const BitField = require('bitfield').default
 const prettyBytes = require('prettier-bytes')
+const path = require('path')
+const fs = require('fs')
```
Comment (DiegoRBaquero) [inline src/renderer/pages/player-page.js:5]: why are these required?

Comment (lavalleeale) [reply src/renderer/pages/player-page.js]: Sorry about that, I was reviewing the possibility of using the torrents image in the `mediaSession`, but removed it after I encountered issues

**Review by DiegoRBaquero — APPROVED** [review]
